07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,173 @@
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                    <ul class="navbar-nav ms-auto">
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Features</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Screenshots</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Team</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Pricing</a>
+                        </li>
+                        <li class="nav-item dropdown">
+                            <a class="nav-link dropdown-toggle dropdown-toggle-split" href="#" role="button"
+                                data-bs-toggle="dropdown" aria-expanded="false">
+                                Drop Down
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a class="dropdown-item" href="#">Action</a></li>
+                                <li><a class="dropdown-item" href="#">Another action</a></li>
+                                <li><a class="dropdown-item" href="#">Something else here</a></li>
+                            </ul>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Pricing</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <!-- Main -->
+    <main class="pb-5 mb-5">
+        <section class="intro py-5">
+            <div class="container px-5">
+                <h1 class="text-center fw-light pt-5">Simple Systeme Fordiscount</h1>
+                <p class="fw-light text-center">Integer Cursus Bibendum Augue Ac Curcus.</p>
+                <div class="intro-details py-5 d-flex flex-column flex-lg-row gap-5">
+                    <div class="detail-1 d-flex flex-column justify-content-center align-items-center">
+                        <img src="assets/images/cloud.svg" class="mt-4 mb-4" alt="" width="50">
+                        <h6>INTRODUCING WHATSAPP</h6>
+                        <p class="text-center small w-75 py-2">Lorem ipsum is simply dummy text of the printing and
+                            typesetting industry</p>
+                        <a href="#" class="small link-dark fw-semibold text-decoration-none">Read More</a>
+                    </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)